es-base-code/main/default/aura/openRecordAction/openRecordActionController.js

```diff
@@ -18,6 +17,7 @@
                 });
             })
             .catch(function (error) {
+                // eslint-disable-next-line no-console
                 console.log(error);
```

Comment:
Let's change that to `console.error` for now. We'll need to do proper error handling later.